app/models/suspended_email.rb

```diff
@@ -16,11 +16,11 @@ def create_from_email_address!(email_address)
     end
 
     def find_with_email(email)
```

Comment:
ditto about clarifying return types
```suggestion
    # @return [SuspendedEmail,nil]
    def find_with_email(email)
```